apps/src/lib/kits/maker/boards/microBit/ExternalLed.js

```diff
@@ -19,4 +18,13 @@ export default class ExternalLed {
   toggle() {
     return this.isOn ? this.off() : this.on();
   }
+
+  blink(delay) {
+    this.off();
+    setInterval(this.toggle_.bind(this), delay);
+  }
```

Comment:
Out of curiosity, how does blink stop? Or is there a way to turn it off?

Reply:
Great question and good catch! There wasn't a way to turn it off, but I just pushed a commit that turns `blink` off when `off` is called. This is how to turn `blink` off for the Circuit Playground.
I'll include a video of this working. Thanks so much!